src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -409,5 +409,46 @@ internal bool IsAwaitStatementContext(int position, CancellationToken cancellati
 
             return false;
         }
+
+        /// <summary>
+        /// Determines whether await should be suggested in a given position.
+        /// </summary>
+        internal bool IsAwaitKeywordContext(int position)
```

Comment:
was this just a move?

Reply:
@CyrusNajmabadi Yep, this was moved from `AwaitKeywordRecommender`.